AddRoutineView.swift

```diff
@@ -0,0 +1,163 @@
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a title", text: $title)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            
+
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a description", text: $description)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 70)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a category", text: $category)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a frequency", value: $frequency, format: .number)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add dates", text: $dates)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
+            
+            Spacer()
+            
+            ZStack {
+                Rectangle()
+                    .foregroundStyle(Color(red: 0.45, green: 0.85, blue: 0.55))
+                
+                TextField("Add a streaklength", value: $streakLength, format: .number)
+                    .foregroundStyle(.black)
+                    .padding([.leading, .top])
+                    .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .frame(width: 360, height: 20)
```

Comment:
The user does not specify a streak length. 

It starts at 0 and is automatically incremented by the app.